src/tokenizer/simple_tokenizer.rs

```diff
@@ -2,7 +2,8 @@ use std::str::CharIndices;
 
 use super::{Token, TokenStream, Tokenizer};
 
-/// Tokenize the text by splitting on whitespaces and punctuation.
+/// Tokenize the text by returning only tokens of consecutive
```

Comment:
While this more exact, I think the original does better explain the intent. Maybe keep both? The old version to rely the intent and define it exactly using the new wording?